terraform/modules/cloudfront/cloudfront.tf

```diff
@@ -108,7 +108,7 @@ resource "aws_cloudfront_distribution" "default" {
 
     forwarded_values = {
       query_string = true
-      headers      = ["Host"]
+      headers      = ["Host", "Authorization"]
```

Comment:
This makes it easier to test on non-prod environments, and shouldn't make a different to prod as no auth headers are used.